src/asset/asset-registry.js

```diff
@@ -480,184 +472,127 @@ Object.assign(pc, function () {
                 filename: filename || name,
                 url: url
             };
-            var data = {};
 
             var asset = self.getByUrl(url);
             if (!asset) {
-                asset = new pc.Asset(name, type, file, data);
+                asset = new pc.Asset(name, type, file, {});
```

Comment:
You don't need to pass the empty object for the last parameter here.

Reply:
The API ref is actually wrong. The `data` parameter should be marked as optional.

Reply:
Thanks Will, I updated the code and the param jsdoc.